framework/platform/bpl/linux/bpl_cfg.cpp

```diff
@@ -28,6 +28,8 @@ using namespace beerocks::bpl;
 
 const char *s_error_strings[] = {FOREACH_ERROR_CODE(GENERATE_ERROR_STRING)};
 
+namespace bpl {
```

Comment:
Shouldn't the s_error_strings above also be in the bpl namespace? Hm, actually, it doesn't even seem to be used...

Reply:
I'll remove this variable.